website/docs/r/data_protection_backup_policy_postgresql.html.markdown

```diff
@@ -0,0 +1,154 @@
+
+* `backup_rules` - (Required) A `backup_rules` block as defined below. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `default_retention_duration` - (Required) The duration of default retention rule. It should follow `ISO 8601` duration format. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+---
+
+* `retention_rules` - (Optional) One or more `retention_rules` blocks as defined below. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+---
+
+A `backup_rules` block supports the following:
+
+* `name` - (Required) The name which should be used for this backup rule. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `repeating_time_intervals` - (Required) Specifies a list of repeating time interval. It should follow `ISO 8601` repeating time interval . Changing this forces a new Backup Policy Postgre Sql to be created.
+
+---
+
+A `retention_rules` block supports the following:
+
+* `name` - (Required) The name which should be used for this retention rule. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `duration` - (Required) Duration of deletion after given timespan. It should follow `ISO 8601` duration format. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `tagging_criteria` - (Required) A `tagging_criteria` block as defined below. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `tagging_priority` - (Required) Retention Tag priority. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+---
+
+A `tagging_criteria` block supports the following:
+
+* `absolute_criteria` - (Optional) Possible values are `AllBackup`, `FirstOfDay`, `FirstOfWeek`, `FirstOfMonth` and `FirstOfYear`. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `days_of_month` - (Optional) One or more `days_of_month` blocks as defined above. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `days_of_the_week` - (Optional) Possible values are `Monday`, `Tuesday`, `Thursday`, `Friday`, `Saturday` and `Sunday`. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `months_of_year` - (Optional) Possible values are `January`, `February`, `March`, `April`, `May`, `June`, `July`, `August`, `September`, `October`, `November` and `December`. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `schedule_times` - (Optional) Specifies a list of schedule times for backup. It should follow `RFC3339` time format. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+* `weeks_of_the_month` - (Optional) Possible values are `First`, `Second`, `Third`, `Fourth` and `Last`. Changing this forces a new Backup Policy Postgre Sql to be created.
+
+---
+
+A `days_of_month` block is day of the month from 1 to 28 otherwise last of month, it supports the following:
```

Comment:
28? what about months that have 30 days?

Reply:
days_of_month is not used to backup postgresql anymore in portal, I'll remove it.